.github/workflows/python-ci.yml

```diff
@@ -174,12 +174,14 @@ jobs:
           python -m pip install --upgrade pip
           pip install tox
 
-      - name: Set LOOKERSDK_CLIENT_{ID,SECRET}
-        # api client/secret stored as github secret per release, e.g. LOOKERSDK_CLIENT_ID__7_20
+      - name: Determine credentials version
         run: |
-          echo "LOOKERSDK_CLIENT_ID=${{ secrets[format('LOOKERSDK_CLIENT_ID__{0}', matrix.looker)] }}" >> $GITHUB_ENV
-          echo "LOOKERSDK_CLIENT_SECRET=${{ secrets[format('LOOKERSDK_CLIENT_SECRET__{0}', matrix.looker)] }}" >> $GITHUB_ENV
+          echo "CREDENTIALS_VERSION=$(${{ github.workspace }}/.github/scripts/parse_version.bash <<< ${{ matrix.looker }})" >> $GITHUB_ENV
```

Comment:
maybe a quick comment as to why we even need to worry about secret versions and a TODO to get rid of them once we remove `21_18` as a supported matrix version?

Reply:
done